### PR TITLE
Avatars

### PR DESCRIPTION
**Pull request by iamalhera**

### CODE REVIEW

File: src/components/avatar/avatar.html [hunk 1]
```diff
@@ -8,15 +8,77 @@
...
+            </div>
+
+            <div class="container-row container-avatar">
+            <img src="/src/assets/alhera-smiling-pic.jpeg" alt="avatar" class="avatar-lg img-rounded">
+            <img src="/src/assets/alhera-2.jpg" alt="avatar" class="avatar-md img-rounded">
+            <img src="/src/assets/alhera-3.jpg" alt="avatar" class="avatar-sm img-rounded">
+            </div>
+
+        </section>
+        <section class="bg-ow">
```
Comment (shahbazibnafzal) [inline src/components/avatar/avatar.html]: class name can be more expressive

**Review by shahbazibnafzal — APPROVED** [review]
You may consider keeping class names a bit more expressive, rest everything looks good.

File: src/components/avatar/avatar.html (hunk 1 above)
Comment (iamalhera) [reply src/components/avatar/avatar.html]: thanks for the suggestion

File: src/components/avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    border-radius: 50%;
+}
+
+.text-avatar{
+    background-color : var(--ca-primary-color-lighter) ;
+    display : flex ;
+    align-items: center;
+    justify-content: center;
+    color : white ;
+    font-size : 2rem ;
```
Comment (Ankur9669) [inline src/components/avatar/avatar.css:30]: Can use color variables if you want

File: src/components/avatar/avatar.html [hunk 3]
```diff
@@ -8,15 +8,77 @@
...
+            <div class="section-description ">
+                Avatars are available in three different sizes. Large avatar, Medium avatar and Small avatar. Here img-rounded applies 50% of border radius to avatar. You can use these according to your need.
+            </div>
+
+            <div class="container-row container-avatar">
+            <img src="/src/assets/alhera-smiling-pic.jpeg" alt="avatar" class="avatar-lg img-rounded">
+            <img src="/src/assets/alhera-2.jpg" alt="avatar" class="avatar-md img-rounded">
+            <img src="/src/assets/alhera-3.jpg" alt="avatar" class="avatar-sm img-rounded">
+            </div>
+
```
Comment (Ankur9669) [inline src/components/avatar/avatar.html]: img tags don't have closing / and indentation could be improved

File: styles.css [hunk 4]
```diff
@@ -1,17 +1,23 @@
...
+    display:flex;
+    height : 100vh ;
+    align-items: center;
+    justify-content: center;
+} 
 .landing-page-button button{
     padding: 1rem;
     border:none;
     background: #762D8B;
     color : white ;
```
Comment (Ankur9669) [inline styles.css]: Here you should use css variables

**Review by Ankur9669 — APPROVED** [review]
The UI and code looks awesome!!!
Just a suggestion you have not added code-examples

**Pull request merged**